test/runtests.jl

```diff
@@ -261,4 +253,70 @@ end
 
       @test cal_energy(sol)/(x[1]-x0[1]+x[2]-x0[2]) ≈ 1e5
    end
+
+   @testset "normalized field" begin
+      # 3D
+      x = range(-10, 10, length=15)
+      y = range(-10, 10, length=20)
+      z = range(-10, 10, length=25)
+      B = fill(0.0, 3, length(x), length(y), length(z)) # [B₀]
+      E = fill(0.0, 3, length(x), length(y), length(z)) # [v₀B₀]
```

Comment:
The symbol $v_0$ is equal to 1m/s? I think the unit of E is [$B_0\cdot m/s$]

Reply:
No it should be $v₀ = l₀/t₀$ that depends on the length and time scale. The unit of $\mathbf{E}$ should be the same as $\mathbf{v}\times\mathbf{B}$, i.e. $v₀B₀$. I've added some new comments and removed the wrong ones.

Reply:
I understand. This unit of $\mathbf{x}$ and $\mathbf{v}$ are all changed.